data-structures/queue/test_queue.py

```diff
@@ -0,0 +1,34 @@
+from queue import Queue
+import pytest
+
+new_queue = Queue()
```

Comment:
Instead of manipulating a global like this, you should be using a fixture for your tests to ensure that each test is independent. Otherwise, one test failing could cause other tests to fail even if you are testing unrelated things.